x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/jira/jira_params.tsx

```diff
@@ -196,191 +196,188 @@ const JiraParamsFields: React.FunctionComponent<ActionParamsProps<JiraActionPara
 
   return (
     <>
+      <EuiFormRow
```

Comment:
This is not as bad as it looks. I remove an extra `<></>`. Please review with "Hide whitespace" enabled.